### ISSUE TITLE
Are we missing a bit in documentation?

### ISSUE THREAD

**Issue by fzyzcjy** [fzyzcjy/flutter_rust_bridge#349]
I am improving the doc and see my old paragraph:

> ### The `mod`
> If you are adding this lib to your own existing code, please put `mod generated_wire;` (where `generated_wire` is the name of the wire file that you choose) into your `lib.rs` or `main.rs`. Only by doing this, Rust can understand that this generated file is a part of your project.

But in https://fzyzcjy.github.io/flutter_rust_bridge/integrate.html seems that this is not discussed. Shall we add this?

/cc @Desdaemon

**Comment by fzyzcjy** [comment]
Another possible problem (?) in documentation (so I reopen this issue to avoid creating too many issues): 

https://fzyzcjy.github.io/flutter_rust_bridge/tutorial_with_flutter.html#macos has a really long description for macos:

![image](https://user-images.githubusercontent.com/5236035/154777786-62cff17e-e032-43cc-8f3e-0664fc9360f2.png)

Maybe we can simplify it now to even a few lines? Because the CI https://github.com/fzyzcjy/flutter_rust_bridge/blob/b905401a157a9291cf3f82c1633e628605866101/.github/workflows/ci.yaml#L53 now is very simple and can work (with the #351 ).

@Desdaemon What do you think? (You are definitely an expert in this area)

**Comment by Desdaemon** [comment]
@fzyzcjy Yes to both. We can leave a link to the related sections or we can have the quick-start point directly to the user guide and remove the old guides.
